### PR TITLE
{chem,math}[foss/2022a,foss/2022b] molmod v1.4.8, TAMkin v1.2.6

### PR DESCRIPTION
**Pull request by Flamefire**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/m/molmod/molmod-1.4.8-gfbf-2022b.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    ('Python', '3.10.8'),
+    ('matplotlib', '3.7.0'),
+]
+
+use_pip = True
+download_dep_fail = True
+sanity_pip_check = True
+
+runtest = "export MATPLOTLIBRC=$PWD; echo 'backend: agg' > $MATPLOTLIBRC/matplotlibrc;"
+runtest += "python setup.py build_ext -i; pytest -ra"
```
[the remark this answers is not in the document]
Comment (jfgrimm) [reply easybuild/easyconfigs/m/molmod/molmod-1.4.8-gfbf-2022b.eb:25]: assuming the reasoning is the same, would be good to copy the comment to the `molmod` easyconfigs as well

[the remark this answers is not in the document]
Comment (Flamefire) [reply easybuild/easyconfigs/m/molmod/molmod-1.4.8-gfbf-2022b.eb:25]: I just copied the EC from the older toolchain, so I don't know. But likely it is the same.

**Comment by Flamefire** [comment]
Test report by @Flamefire
**FAILED**
Build succeeded for 3 out of 5 (5 easyconfigs in total)
n1170 - Linux RHEL 8.7 (Ootpa), x86_64, Intel(R) Xeon(R) Platinum 8470 (icelake), Python 3.8.13
See https://gist.github.com/Flamefire/23d5232d776586d23c5d9397cf75f10a for a full test report.

**Comment by Flamefire** [comment]
Test report by @Flamefire
**FAILED**
Build succeeded for 4 out of 5 (5 easyconfigs in total)
n1170 - Linux RHEL 8.7 (Ootpa), x86_64, Intel(R) Xeon(R) Platinum 8470 (icelake), Python 3.8.13
See https://gist.github.com/Flamefire/e5900af89e4dd363a5a24b3e7107226a for a full test report.

**Comment by Flamefire** [comment]
Test report by @Flamefire
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in total)
n1170 - Linux RHEL 8.7 (Ootpa), x86_64, Intel(R) Xeon(R) Platinum 8470 (icelake), Python 3.8.13
See https://gist.github.com/Flamefire/1faa7eb3e33944ffc3db91cb8084443e for a full test report.

**Comment by Flamefire** [comment]
Test report by @Flamefire
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in total)
n1170 - Linux RHEL 8.7 (Ootpa), x86_64, Intel(R) Xeon(R) Platinum 8470 (icelake), Python 3.8.13
See https://gist.github.com/Flamefire/5cfa9373ce4231b485ba2c3a1aa23f58 for a full test report.

**Comment by jfgrimm** [comment]
@boegelbot: please test @ jsc-zen3

**Comment by jfgrimm** [comment]
@boegelbot: please test @ generoso

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**FAILED**
Build succeeded for 6 out of 8 (5 easyconfigs in total)
node107.viking2.yor.alces.network - Linux Rocky Linux 8.8, x86_64, AMD EPYC 7643 48-Core Processor, Python 3.6.8
See https://gist.github.com/jfgrimm/125bbb5526a3f605c4851ce23c1cb5bf for a full test report.

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in total)
node107.viking2.yor.alces.network - Linux Rocky Linux 8.8, x86_64, AMD EPYC 7643 48-Core Processor, Python 3.6.8
See https://gist.github.com/jfgrimm/af39c138bfb0d98d6e9f4578d33e01f4 for a full test report.

**Comment by jfgrimm** [comment]
same test failure on generoso for both `TAMkin` easyconfigs:

```
======================================================================
FAIL: test_vsa_no_mass (tamkin.test.test_nma.NMATestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/tmp/boegelbot/TAMkin/1.2.6/foss-2022a/tamkin-1.2.6/tamkin/test/test_nma.py", line 532, in test_vsa_no_mass
    self.check_freqs(expected_freqs, nma, 0, check_zeros=True)
  File "/tmp/boegelbot/TAMkin/1.2.6/foss-2022a/tamkin-1.2.6/tamkin/test/test_nma.py", line 77, in check_freqs
    self.assertAlmostEqual(
AssertionError: 0.13774798 != 0.6460767049351539 within 0 places (0.5083287249351539 difference) : Frequency 2 does not match, expected - computed = 0.1 - 0.6 = -0.508329
```

**Comment by Flamefire** [comment]
> same test failure on generoso for both `TAMkin` easyconfigs:

And same as for all on #19722   
This is https://github.com/molmod/tamkin/issues/43

It only seems to happen on haswell. But I don't have access to a haswell machine anymore so can't actually test

**Comment by Flamefire** [comment]
Test report by @Flamefire
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in total)
i8014 - Linux Rocky Linux 8.7, x86_64, AMD EPYC 7352 24-Core Processor (zen2), 8 x NVIDIA NVIDIA A100-SXM4-40GB, 545.23.08, Python 3.6.8
See https://gist.github.com/Flamefire/58d31f94cf0846d1bc51c523f9a7cb79 for a full test report.

**Comment by Flamefire** [comment]
Test report by @Flamefire
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in total)
n1358 - Linux RHEL 8.7 (Ootpa), x86_64, Intel(R) Xeon(R) Platinum 8470 (icelake), Python 3.8.13
See https://gist.github.com/Flamefire/f12f08cce07236235aaa4a6bfb104e03 for a full test report.

**Comment by Flamefire** [comment]
@jfgrimm Please try this and #17222 again. It should pass the test now.

**Comment by Flamefire** [comment]
@jfgrimm or @casparvl This PR is still pending. The other TAMkin PRs have been merged, only 2022x toolchain versions here are missing.

**Comment by ocaisa** [comment]
@boegelbot: please test @ generoso

**Comment by ocaisa** [comment]
@boegelbot: please test @ jsc-zen3

**Comment by ocaisa** [comment]
@boegelbot: please test @ jsc-zen3

**Comment by Flamefire** [comment]
@ocaisa Can you merge this? Reports are green

**Review by ocaisa — APPROVED** [review]
I'm on holidays today but...

**Pull request merged**